cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -61,7 +61,7 @@ func CreateOrUpdate(ws, deploymentID, deploymentName, name, action, workerType s
 	}
 
 	if requestedDeployment.Id == "" {
-		fmt.Printf("No Deployments found in workspace %s\n", ansi.Bold(ws))
+		fmt.Printf("%s %s\n", deployment.NoDeploymentInWSMsg, ansi.Bold(ws))
```

Comment:
🤔 log a ticket to properly test worker queue flows?

Reply:
For context, GH show an indication this change is not covered by any tests. This is not something you have done in this PR but clearly an existing test gap.